indexd/index/drivers/alchemy.py

```diff
@@ -721,9 +720,11 @@ def get_latest_version(self, did):
                 raise MultipleRecordsFound('multiple records found')
 
             query = session.query(IndexRecord)
-            record = query.filter(IndexRecord.baseid == baseid) \
-                .order_by(IndexRecord.created_date.desc()).first()
-
+            query = query.filter(IndexRecord.baseid == baseid) \
```

Comment:
Isn't the first query unnecessary if has_version is true? Its just going to get overwritten.
Also kinda hard to tell with 5 assignments to `query` in one function